src/crud/session.py

```
@@ -303,11 +331,56 @@ async def update_session(
     return honcho_session
 
 
+async def _batch_delete_matching(
+    db: AsyncSession,
+    model: Any,
```

Ruff (0.14.7)

Dynamically typed expressions (typing.Any) are disallowed in `model`

(ANN401)